custom-plugins/grafana-worldmap-panel/src/worldmap.ts

```diff
@@ -150,7 +160,12 @@ export default class WorldMap {
       if (circle) {
         circle.setRadius(this.calcCircleSize(dataPoint.value || 0));
         circle.setStyle({
-          color: this.getColor(dataPoint.value),
+          radius:
+            selectedFacilityName === dataPoint.locationName
+              ? this.calcCircleSize(dataPoint.value || 0) * 1.5
+              : this.calcCircleSize(dataPoint.value || 0),
+          weight: selectedFacilityName === dataPoint.locationName ? 5 : 0.5,
+          color: selectedFacilityName === dataPoint.locationName ? 'grey' : this.getColor(dataPoint.value),
```

Comment:
hadn't thought of that before.. either spelling works 'gray' or 'grey'! ( I prefer to use hex values )

Reply:
Yeah, this was before me -- I just left it as is.